app/views/appointments/advisors.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container ">
+<br></br><h1 class="display-5 fw-bold" align = "center">Create an Appointment with a Counselor</h1><br></br>
+    <div class="row">
+      <div class ="col-sm">
+        <p class="lead mb-4" align="center">Amanda</p>
+        <iframe class="col-lg-12 col-md-12 col-sm-12" src="https://calendar.google.com/calendar/u/0/selfsched?sstoken=UUFTcERNeEp2clFNfGRlZmF1bHR8NGE3NmY4OTQ2ZWE5MTM3MTViYzM0ZWE1YTM3ZjE1NWU" height="500" width="300" align = "left">
+            this is an iframe
+        </iframe>
+      </div>
+      <div class="col-sm">
+      <p class="lead mb-4" align="center">Trinh</p>
```

Comment:
<img width="301" alt="Screen Shot 2022-04-14 at 3 52 59 PM" src="https://user-images.githubusercontent.com/69605001/163490394-22285d5e-a5f4-4e2d-933f-93e9fdec2990.png">
On mobile this page looks a bit disjointed; since our app is mobile-first can we clean up some styling here? Specifically, add a little bit of spacing between the first iframe & "Trinh" and also center the iframes if possible. 

Additionally, I'm not sure if this can be totally fixed, but can you try to horizontally extend the iframes, because dates on the Gcal iframe are overlapping and difficult to read. If you could resize them to minimize the overlap that would be great.